### PR TITLE
docs(readme): add relevant readme instructions; fix npm commands

### PR DESCRIPTION
**Pull request by cerebrl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,120 @@
...
+**This is a prototype of a development framework for integrating a ForgeRock Login Widget into an existing app, or running as a standalone Login App. This project is not officially supported and is not recommended for any project development. If you use this, you accept all the risks that come with unsupported software.**
 
-If you're seeing this, you've probably already done this step. Congrats!
+## Technical Requirements
 
-```bash
-# create a new project in the current directory
-npm init svelte@next
+1. Node.js v16 or higher
+2. npm v8 or higher
```
Comment (ryanbas21) [inline README.md:10]: Lets add an `engines` property to package.json enforcing this, or a `.nvmrc` file?

**Review by ryanbas21 — COMMENTED** [review]
Just a comment about engines

**Comment by ryanbas21** [comment]
tests/widget.test.js looks to have broken ci.

File: README.md (hunk 1 above)
Comment (cerebrl) [reply README.md:10]: Added to the `engines` property. Thanks!

**Review by ryanbas21 — APPROVED** [review]

**Pull request merged**